data/bootstrap_hwdefinition.yaml

```diff
@@ -0,0 +1,65 @@
+---
+#############################################################################################################################
```

Comment:
Good idea, I'll edit the headers for both files. Any specific headers required for AT&T produced open source?